addons/l10n_in_sale/models/sale_order.py

```diff
@@ -9,8 +9,25 @@ class SaleOrder(models.Model):
 
     l10n_in_reseller_partner_id = fields.Many2one('res.partner',
         string='Reseller', domain="[('vat', '!=', False), '|', ('company_id', '=', False), ('company_id', '=', company_id)]", states={'posted': [('readonly', True)]})
+    l10n_in_journal_id = fields.Many2one('account.journal', string="Journal", states={'posted': [('readonly', True)]})
 
     def _prepare_invoice(self):
         invoice_vals = super(SaleOrder, self)._prepare_invoice()
-        invoice_vals['l10n_in_reseller_partner_id'] = self.l10n_in_reseller_partner_id.id
```

Comment:
need to check we don't remove this (nothing to do with multiple operational units)